catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/EntityResourceTest.java

```diff
@@ -793,6 +840,47 @@ void put_addDeleteInvalidFollower_200(TestInfo test) throws IOException, URISynt
   ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
   // Common entity tests for PATCH operations
   ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+  @Test
+  void patch_entityDescriptionAndTestAuthorizer(TestInfo test) throws IOException, URISyntaxException {
+    if (!supportsPatch || !supportsAuthorizedMetadataOperations) {
+      return;
+    }
+
+    T entity = createEntity(createRequest(getEntityName(test), "description", null, null), adminAuthHeaders());
+
+    // Anyone can update description on unowned entity.
+    entity = patchEntityAndCheckAuthorization(getEntityInterface(entity), TestUtils.ADMIN_USER_NAME, false);
+    entity = patchEntityAndCheckAuthorization(getEntityInterface(entity), USER1.getName(), false);
+    entity = patchEntityAndCheckAuthorization(getEntityInterface(entity), USER_WITH_DATA_STEWARD_ROLE.getName(), false);
+    entity =
+        patchEntityAndCheckAuthorization(getEntityInterface(entity), USER_WITH_DATA_CONSUMER_ROLE.getName(), false);
+
+    EntityInterface<T> entityInterface = getEntityInterface(entity);
+
+    if (!supportsOwner) {
+      return;
+    }
+
+    // Set the owner for the table.
+    String originalJson = JsonUtils.pojoToJson(entity);
+    ChangeDescription change = getChangeDescription(entityInterface.getVersion());
+    change.getFieldsAdded().add(new FieldChange().withName("owner").withNewValue(USER_OWNER1));
+    entityInterface.setOwner(USER_OWNER1);
+    entity =
+        patchEntityAndCheck(
+            entityInterface.getEntity(),
+            originalJson,
+            authHeaders(USER1.getName() + "@open-metadata.org"),
+            MINOR_UPDATE,
+            change);
+
+    // Admin, owner (USER1) and user with DataSteward role can update description on entity owned by USER1.
+    entity = patchEntityAndCheckAuthorization(getEntityInterface(entity), TestUtils.ADMIN_USER_NAME, false);
+    entity = patchEntityAndCheckAuthorization(getEntityInterface(entity), USER1.getName(), false);
+    entity = patchEntityAndCheckAuthorization(getEntityInterface(entity), USER_WITH_DATA_STEWARD_ROLE.getName(), false);
+    patchEntityAndCheckAuthorization(getEntityInterface(entity), USER_WITH_DATA_CONSUMER_ROLE.getName(), true);
+  }
+
   @Test
   void patch_entityAttributes_200_ok(TestInfo test) throws IOException, URISyntaxException {
```

Comment:
These tests remain unchanged and prove that the change introduced in this PR is backward compatible.